src/viur/core/bones/uri.py

```diff
@@ -45,10 +42,13 @@ def __init__(
         :param local_path_allowed: If True, the URLs that are local paths will be prefixed with "/".
         """
         super().__init__(**kwargs)
-        self.accepted_ports = set(sorted(UriBone._build_accepted_ports(accepted_ports), key=lambda rng: rng.start))
+        if accepted_ports:
+            self.accepted_ports = set(sorted(UriBone._build_accepted_ports(accepted_ports), key=lambda rng: rng.start))
```

Comment:
The idea here was to have the ports sorted once to provide them in a friendly readable way for the structure.

```suggestion
            self.accepted_ports = sorted(set(UriBone._build_accepted_ports(accepted_ports)), key=lambda rng: rng.start)
```